### PR TITLE
3.0.0

### PR DESCRIPTION
**Pull request by nt4f04uNd**

### CODE REVIEW

File: example/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+   "dart.flutterSdkPath": "D:\\dev\\src\\fvm\\versions\\2.10.3"
```
Comment (Abestanis) [inline example/.vscode/settings.json:2]: I'm not sure if you really want to commit this with a local path.

**Review by Abestanis — COMMENTED** [review]
Do you think it would be good to also remove `NFBackButtonListener` in this release, because the issue mentioned in the comment above the class has been fixed.

**Comment by nt4f04uNd** [comment]
> Do you think it would be good to also remove NFBackButtonListener in this release, because the issue mentioned in the comment above the class has been fixed.

Good catch, deleted it

**Comment by Abestanis** [comment]
@nt4f04uNd I saw 0b65c248337024da6c456ff984ffe73b8bbe4dc2, are you working on an update to Flutter 3 for Sweyer as well? We have to rename the `MenuItem` class because it clashes with another class from the Material library. If you are not working on it, I would create a quick pull request.

**Comment by nt4f04uNd** [comment]
Yes, this was on my radar, I only made it compilable with Flutter 3 for now, but I want to fully migrate it now that I merged https://github.com/nt4f04uNd/sweyer/pull/34

**Comment by nt4f04uNd** [comment]
Merging, since Sweyer 1.0.8 is now out

**Pull request merged**